nupic/research/frameworks/pytorch/models/resnets.py

```diff
@@ -134,25 +127,23 @@ def conv_layer(
 
 def activation_layer(
     out,
-    percent_on,
-    boost_strength,
-    boost_strength_factor,
-    k_inference_factor,
-    local,
-    *args,
+    params,
 ):
     """Basic activation layer.
     Defaults to ReLU if percent_on is < 0.5. Otherwise KWinners is used."""
-    if percent_on >= 0.5:
+    if params.params_function is not None:
+        params = params.params_function(0, out, 0)
```

Comment:
Same comment as before. It looks like LayerParams should be implemented as a class